src/TransactionTestsFiller/ttGasLimit/TransactionWithGasLimitxPriceOverflowFiller.json

```diff
@@ -1,15 +1,13 @@
 {
-    "TransactionWithGasLimitxPriceOverflow" : {
-	     "expect" : [
-            {
-                "network" : [">=Frontier"],
-                "result" : "valid"
-            }
-        ],
+    "TransactionWithGasLimitxPriceOverflow" :
+    {
+	    "expectException" : 
+        {
+        },
         "transaction" :
         {
             "data" : "",
-            "gasLimit" : "115792089237316195423570985008687907853269984665640564039457584007913129639935",
```

Comment:
the goal of this test is to make   gasLimit * gasPrice  >  2**256-1, need to change. is there a cap on gasPrice?

Reply:
I am changing this test to be gasLimit=0xffffffffffffffff, gasPrice=0x1000000000000000100000000000000010000000000000002, where gasLimit is the highest uint64 value, and gasPrice is the lowest value such as the product is equal/higher than a 256bit value.
The test still does not produce an exception.

Reply:
Well the transaction technically valid.
Just that irl no account with enough balance exists.

Is int64max a limit of gasLimit field?

Reply:
what is the exception? GasLimitPriceProductOverflow